test/ServerTest.php

```diff
@@ -253,6 +254,16 @@ public function testListenPassesCallableArgumentToCallback()
             ->method('getHeaders')
             ->will($this->returnValue([]));
 
+        $responseBody = new Stream('php://temp');
+        $this->response
+            ->expects($this->any())
+            ->method('getBody')
+            ->willReturn($responseBody);
+        $this->response
+            ->expects($this->any())
+            ->method('withHeader')
+            ->willReturnSelf();
+
```

Comment:
With some of these integration tests, it seems like the only way to do it. They do get lengthy, though!

We might consider porting them to Prophecy at some point (we're using it extensively in the ZF3 component rewrites, and in Expressive), but that's something for a lull time in development.